src/chrome/content/rules/Rooster-Teeth.xml

```diff
@@ -0,0 +1,67 @@
+<!--
+	Unused domains:
+		calendar.roosterteeth.com
+		elr.roosterteeth.com
+		www.rfb.roosterteeth.com
+		www.rgb.roosterteeth.com
+		rvb.roosterteeth.com
+		s3.roosterteeth.com
+		securetransfer.roosterteeth.com
+		staffstore.roosterteeth.com
+		upload.roosterteeth.com
+		v2.roosterteeth.com
+		staging.v2.roosterteeth.com
+
+	Time out:
+		email.roosterteeth.com
+		jump.roosterteeth.com
+		email.mg.roosterteeth.com
+
+	Invalid certificate:
+		click.email.roosterteeth.com
+		rtbox.roosterteeth.com
+
+	Development domains:
+		auth.roosterteeth.com
+		business-service.roosterteeth.com
+		dev.roosterteeth.com
+		qa.roosterteeth.com
+		funhaus.qa.roosterteeth.com
+		staging.roosterteeth.com
+		achievementhunter.staging.roosterteeth.com
+		cowchop.staging.roosterteeth.com
+		funhaus.staging.roosterteeth.com
+		gameattack.staging.roosterteeth.com
+		screwattack.staging.roosterteeth.com
+		store.staging.roosterteeth.com
+		theknow.staging.roosterteeth.com
+
+-->
+<ruleset name="Rooster Teeth">
+
+	<target host="roosterteeth.com" />
+	<target host="www.roosterteeth.com" />
+	<target host="accounts.roosterteeth.com" />
```

Comment:
Looks like the host `http://accounts.roosterteeth.com/` has errors. Please add to list of non-functional domains